py_neuromodulation/ConnectivityDecoding/_get_grid_whole_brain.py

```diff
@@ -1,18 +1,15 @@
 import nibabel as nib
 import numpy as np
-import scipy.io as sio
-import os
 from matplotlib import pyplot as plt
 
-
 class NiiToMNI:
 
     def __init__(
         self,
         PATH_nii_file: str = r"C:\code\RMap_ROI_Estimation\Automated Anatomical Labeling 3 (Rolls 2020).nii",
     ) -> None:
 
-        self.img = nib.load(PATH_nii_file)
+        self.img = nib.Nifti1Image.from_filename(PATH_nii_file)
```

Comment:
I think those files need to be restructured in further detail by me. I would for the mypy validation leave out the whole ConnectivityDecoding folder for now.
But thanks for addressing those also already.